src/map.js

```diff
@@ -126,13 +112,6 @@ export default function chloro_map(data1, data2, error) {
       return d ? "M" + d.join("L") + "Z" : null;
     });
 
-  function groupBy(data2, accessor) {
```

Comment:
the big error was that you were passing in data2 which had a form like {origin: {0: ..}, ..} rather than [{origin: x, ..}, {origin: y,..}, ..], so i moved that to right after you load the data so you won't be able to make that mistake again

Reply:
also there's a bunch of changes, which are mostly automformatting